pay-api/src/pay_api/utils/converter.py

```diff
@@ -26,6 +29,11 @@ def __init__(
         self.register_structure_hook(Decimal, self._structure_decimal)
         self.register_unstructure_hook(Decimal, self._unstructure_decimal)
         self.register_unstructure_hook(datetime, self._unstructure_datetime)
+        self.register_structure_hook(datetime, self._structure_datetime())
```

Comment:
Will this impact others hmm might want to look at that

Reply:
removed